### DISCUSSION TITLE
syntax color changed in Trusted sites pane with the latest dev build

### DISCUSSION THREAD

**Discussion by uBlock-user** [uBlockOrigin/uBlock-issues#2521] [General] [answered]
![Capture2](https://user-images.githubusercontent.com/21290713/222339901-e484eee5-7681-4223-85b1-7860cb330d95.JPG)

The above is with the 3b5 build, this was [not the color](https://user-images.githubusercontent.com/21290713/222340076-44bdbd77-14b0-42d4-88e1-0c6aa89f6bda.JPG) till 3b5 build. Something broke with https://github.com/gorhill/uBlock/commit/33c437f99f30daa3172b097ba35ae662239f6013 ?

@gorhill

**Comment by gorhill** [comment, marked as answer]
https://github.com/gorhill/uBlock/commit/a60a5c277227415b3a39b484d60f65220cc207e2
